erpnext/stock/doctype/pick_list/test_pick_list.py

```
@@ -1485,3 +1485,123 @@ def test_pick_list_with_and_without_so(self):
+				pl.cancel()
+			if pl_manual.docstatus == 1:
+				pl_manual.cancel()
+			so.cancel()
+			so2.cancel()
+			se1.cancel()
+			se2.cancel()
+			consume_se.cancel()
+			active_batch.delete()
+			disabled_batch.delete()
+		except Exception:
+			pass
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)